Author: Ole Streicher <debian@liska.ath.cx>
Description: Set the default plugin path to the one where recipes are
 installed in Debian.
--- a/etc/esorex.rc.in
+++ b/etc/esorex.rc.in
@@ -130,7 +130,7 @@
 # as well. Multiple directory heads may be specified, by separating the
 # starting paths with colons (:). This option may also be set using the
 # environment variable ESOREX_PLUGIN_DIR.
-esorex.caller.recipe-dir=@prefix@/lib/esopipes-plugins
+esorex.caller.recipe-dir=@exec_prefix@/lib/@build_alias@/cpl/plugins:@exec_prefix@/lib/cpl/plugins
 
 # --suppress-link
 # When TRUE, no symbolic link is created to the output product. However, if
